src/components/scenes/CoinRankingDetailsScene.tsx

```diff
@@ -447,32 +469,38 @@ const CoinRankingDetailsSceneComponent = (props: Props) => {
         <View style={styles.container}>
           <EdgeAnim style={styles.titleContainer} enter={fadeInLeft}>
             <FastImage style={styles.icon} source={imageUrlObject} />
-            <EdgeText style={styles.title}>{`${currencyName} (${currencyCodeUppercase})`}</EdgeText>
+            <EdgeText style={styles.title}>{`${currencyName} (${currencyCode})`}</EdgeText>
           </EdgeAnim>
-          <SwipeChart assetId={coinRankingData.assetId} currencyCode={currencyCodeUppercase} fiatCurrencyCode={initFiat} />
-          <View style={styles.buttonsContainer}>
-            <IconButton label={lstrings.title_buy} onPress={handleBuyPress}>
-              <Fontello name="buy" size={theme.rem(2)} color={theme.primaryText} />
-            </IconButton>
-            <IconButton label={lstrings.title_sell} onPress={handleSellPress}>
-              <Fontello name="sell" size={theme.rem(2)} color={theme.primaryText} />
-            </IconButton>
-            {isStakingAvailable() && (
-              <IconButton label={getUkCompliantString(countryCode, 'stake_earn_button_label')} onPress={handleStakePress}>
-                <Feather name="percent" size={theme.rem(2)} color={theme.primaryText} />
+          <SwipeChart assetId={coinRankingData.assetId} currencyCode={currencyCode} fiatCurrencyCode={initFiat} />
+          {edgeAssets.length > 0 && (
```

Comment:
Do `edgeAssets.length <= 0 ? null : <View ....>`, since using `&&` to guard rendering will generate an ESLint warning once we turn on strict-bookean.